rust/src/treefile.rs

```diff
@@ -1862,6 +1881,23 @@ pub(crate) mod tests {
         assert!(treefile.base.automatic_version_prefix.unwrap() == "30");
         assert!(treefile.base.mutate_os_release.unwrap() == "30");
         assert!(treefile.base.rpmdb.is_none());
+        let buf = indoc! {r#"
+            variables:
+                releasever: foo
+            releasever: 30
+        "#};
+        let mut input = io::BufReader::new(buf.as_bytes());
```

Comment:
Not that this really matters at all, but for static bytes all we need is [Cursor](https://doc.rust-lang.org/std/io/struct.Cursor.html).

Reply:
Good idea. Let's do a follow-up cleanup where we fix this tree-wide since it's pretty widespread.